pkg/controller/controller_instance.go

```diff
@@ -290,12 +290,12 @@ func (c *controller) reconcileServiceInstanceDelete(instance *v1alpha1.ServiceIn
 		}
 	}
 
-	glog.V(4).Infof("Deprovisioning ServiceInstance %v/%v of ClusterServiceClass %v at ClusterServiceBroker %v", instance.Namespace, instance.Name, serviceClass.Name, brokerName)
+	glog.V(4).Infof(`ServiceInstance "%v/%v": Deprovisioning ClusterServiceClass %v at ClusterServiceBroker %v`, instance.Namespace, instance.Name, serviceClass.Name, brokerName)
 	response, err := brokerClient.DeprovisionInstance(request)
 	if err != nil {
 		if httpErr, ok := osb.IsHTTPError(err); ok {
 			s := fmt.Sprintf(
-				"Error deprovisioning ServiceInstance \"%s/%s\" of ClusterServiceClass %q at ClusterServiceBroker %q with status code %d: ErrorMessage: %v, Description: %v",
+				`ServiceInstance "%v/%v": Error deprovisioning, ClusterServiceClass %q at ClusterServiceBroker %q with status code %d: ErrorMessage: %v, Description: %v`,
```

Comment:
With all of the messages that contain a service class or plan, we should have the k8s names and external names clearly identified.  What I did in #1336 was to use this format:

```
ClusterServiceClass (K8S: %q ExternalName: %q)
```

Reply:
The phrasing seems awkward here. I'd make it something like:

`ServiceInstance "%v/%v": Deprovision call failed; received error response from broker: Status Code: %d, Error Message: %v, Description: %v"`

If you change it to that, you can remove the `"Deprovision call failed. "+` from 315. 😄

Reply:
I agree the wording seems a little awkward; no real need to make it perfect, since 'better' will do for now.

I'm all for another pass where we remove redundant information from conditions and error messages and make things even better :)